jgclark.Summaries/src/summaryHelpers.js

```diff
@@ -720,10 +724,19 @@ export async function generateProgressUpdate(occObjs: Array<TMOccurrences>, peri
     const daysBetween = toDateMom.diff(fromDateMom, 'days')
     // Include sparklines only if this period is a month or less
     const showSparklines = (requestToShowSparklines && daysBetween <= 31)
-    if (showSparklines) {
-      // Truncate lines to avoid line wrapping
-      occObjs.forEach((m) => m.term = truncateString(m.term, 60, 'middle'))
-    }
+
+    await occObjs.forEach(async (m) => {
+      if (m.term.includes('<%') && m.term.includes('%>')) {
+        m.term = await NPTemplating.render(m.term)
```

Comment:
this is returning late

Reply:
I'm not sure what you're trying to achieve here when you say simply "render templates".
Nor do I know the Templating code, and as it's not well documented, it would take me a very long time to figure out. So I can't help. Do you want to back out this part of the PR for now?

Reply:
Looks pretty cool to me. You could include template code in the title. Sounds like it could be useful. @aaronpoweruser What's your use case? And you tested it I assume.

Reply:
@aaronpoweruser Can you pls provide some test examples that show how your new code is used, especially the new template parts? I don't see any mention of the template strings in the README. I may have missed it.

Reply:
Sure, I can update the documentation, this was a for me feature. As I dynamically generate a link to my weekly note.